flink-core/src/main/java/org/apache/flink/util/InstantiationUtil.java

```diff
@@ -91,6 +92,40 @@ public ClassLoaderObjectInputStream(InputStream in, ClassLoader classLoader) thr
 			return super.resolveClass(desc);
 		}
 
+		@Override
+		protected Class<?> resolveProxyClass(String[] interfaces) throws IOException, ClassNotFoundException {
```

Comment:
I'm wondering if a test case can be written for this. Is there a test case for correctly resolving classes in a basic (non-proxy case)?

Reply:
I read the code, it seems that there is no test case for this. I will try to add a test case.